assets/stylesheets/lobby.css

```diff
@@ -400,6 +400,10 @@ p {
     background-color: #d9544f81;
     border-radius: 5px;
 }
+.selected-avatar-3 {
```

Comment:
Could we get an extra space above this pls? :)

Reply:
Yep, also screenshot 
![image](https://user-images.githubusercontent.com/97006153/151692224-d0d0809c-dca3-496a-9838-3b60960b87bd.png)